content/doc/pipeline/tour/post.adoc

```diff
@@ -0,0 +1,109 @@
+---
+layout: documentation
+title: Cleaning up and notifications
+---
+
+Since the `post` section of a Pipeline is guaranteed to run at the end of a
+Pipeline's execution, we can add some notification or other steps to perform
+finalization, notification, or other end-of-Pipeline tasks.
+
+[pipeline]
+----
+// Declarative //
```

Comment:
Also, the tour is not the same as the syntax reference. Without a compelling reason I'm going to request that your request be denied :smiling_imp: